gustaf/utils/connec.py

```diff
@@ -191,10 +191,7 @@ def faces_to_edges(faces):
 
     for i in range(vertices_per_face):
         # v_ind : corresponding vertex index for i value
-        if i == int(vertices_per_face - 1):
-            v_ind = 0
-        else:
-            v_ind = i + 1
+        v_ind = 0 if i == int(vertices_per_face - 1) else i + 1
```

Comment:
Not sure if we want this simplification. Could make it harder to read?

Should be Rule SIM108, i think.

Reply:
Seems fine